.github/mergify.yml

```diff
@@ -0,0 +1,15 @@
+defaults:
+  actions:
+    backport:
+      assignees:
+        - "{{ author }}"
+
+pull_request_rules:
```

Comment:
Should we add `"#approved-reviews-by>=1"` in `queue_rules` as suggested in in mergify [doc](https://docs.mergify.com/merge-queue/setup)?

It's also used by [Osmosis](https://github.com/osmosis-labs/osmosis/blob/6f033c5aad6edf4e07f15e616532986eec55b6da/.github/mergify.yml#L7)

Reply:
We can add a merge queue later, but for now this is just to set up the backport PRs